docs/TestingFrameworks.md

```diff
@@ -0,0 +1,31 @@
+---
+id: testing-frameworks
+title: Testing other frameworks
+layout: docs
+category: Guides
+permalink: docs/testing-frameworks.html
+previous: migration-guide
+next: troubleshooting
+---
+
+Although Jest may be considered React-specific test runner, in fact it is a universal testing platform, with the ability to adapt to any JavaScript library or framework. In this section we'd like to link to community posts and articles about integrating Jest into other popular JS libraries.
+
+## Vue.js
+
+* [Jest for all: Episode 1 — Vue.js](https://medium.com/@kentaromiura_the_js_guy/jest-for-all-episode-1-vue-js-d616bccbe186#.d573vrce2) by Cristian Carlesso ([@kentaromiura](https://twitter.com/kentaromiura))
+
+## AngularJS
+
+* [Testing an AngularJS app with Jest](https://medium.com/aya-experience/testing-an-angularjs-app-with-jest-3029a613251) by Matthieu Lux ([@Swiip](https://twitter.com/Swiip))
+
+## Angular
```

Comment:
Yup, you should see the upgrade guide (because it's possible somehow), it's pure gold XD